src/Microsoft.IdentityModel.Tokens/TokenUtilities.cs

```diff
@@ -55,5 +58,70 @@ internal static IEnumerable<SecurityKey> GetAllSigningKeys(TokenValidationParame
                 foreach (SecurityKey key in validationParameters.IssuerSigningKeys)
                     yield return key;
         }
+
+        /// <summary>
+        /// Merges claims. If a claim with same type exists in both the lists, the one in claims will be kept.
```

Comment:
when referencing parameters it is standard to use <paramref name="symbolName"/> eg `<paramref name="claims"/>`

Reply:
New Learning :)